lab-integration/src/main.ts

```diff
@@ -0,0 +1,24 @@
+import { promises as fs } from 'fs';
+import path from 'path';
+
+import JsonFileTestsManager from './modules/tools/JsonFileTestsManager';
+import { testsResult } from './modules/types';
+
+(async () => {
+  const jsonFileTestsManager = new JsonFileTestsManager(
+    (await fs.readFile(path.join(process.cwd(), '../test-results.json'), {
+      encoding: 'utf8',
+    })) as unknown as testsResult[],
+  );
+
+  console.info('SCRIPT IS RUNNING...');
+
+  setInterval(async () => {
+    const dataset = await jsonFileTestsManager.readJsonTestsResultFile();
+    if (jsonFileTestsManager.isTestsResultsFileUpdated(dataset)) {
+      console.warn('__________________________');
+      console.warn(`[${new Date()}] TESTS RESULTS FILE UPDATED`);
+      console.warn('__________________________');
+    }
+  }, 1000);
```

Comment:
Let's define this delay in a way that we can easily control. Maybe in `.env`? Or the top of this file?